spec/count_words_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rspec'
+require 'count_words'
+
+describe 'initializing counter' do
+  let(:counter) { CountWords.new }
+
+  expected = {}
```

Comment:
It was meant to make all of these tests have a parallel structure.
But, it should be inside the 'it'.